src/buffer/out/textBuffer.cpp

```diff
@@ -3280,7 +3279,7 @@ MarkExtents TextBuffer::_scrollMarkExtentForRow(const til::CoordType rowOffset,
             const auto nextX = gsl::narrow_cast<uint16_t>(x + length);
             const auto markKind{ attr.GetMarkAttributes() };
 
-            if (markKind != MarkKind::None)
+            if (markKind != MarkKind::Output)

```

Comment:
You know, entirely removing this conditional _seems_ to work, form 1 minute of playing with it